api/services/elastigroup/azure/spotVms/paths/groupVmDeploymentDetails.yaml

```diff
@@ -8,7 +8,7 @@ get:
   parameters:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
-    - in: query
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master